packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -49,7 +51,8 @@ export const DefaultCell = (props: TableCellProps) => {
 
   const isStringValue = typeof value === 'string';
 
-  const cellStyle = getCellStyle(tableStyles, cellOptions, displayValue, inspectEnabled, isStringValue);
+  const textShouldWrap = displayValue.text.length < OG_TWEET_LENGTH && displayValue.text.search(/\s/) >= 0;
```

Comment:
To be clear, it **_should_** wrap if it's longer then 140 characters || there's whitespace?

Reply:
I should add a comment for this 😄, it should wrap if the length of displayed text is less than 140 characters and contains whitespace.

So large strings won't wrap (and don't display well this way anyhow) and short strings that have continuous text won't wrap either.